src/Tokenizer/TokensAnalyzer.php

```diff
@@ -525,33 +536,32 @@ public function isWhilePartOfDoWhile($index)
     }
 
     /**
-     * @param Tokens $tokens
-     * @param int    $start
+     * Find classy elements.
      *
-     * @return array
+     * Searches in tokens from the classy index till the end of the classy.
+     * Returns an 2D array; (int) the index until the method has analysed, (array) found classy elements.
+     *
+     * @param int $index classy index
+     *
+     * @return array<int, array>
```

Comment:
This should be `array<int|array>` (or maybe `array` would be enough given the detailed description above).

Reply:
it always return an `array<int, array>` , never `array<int>` or `array<array>` therefore `array<array|int>` would not be correct